.changeset/fast-cows-impress.md

```diff
@@ -0,0 +1,5 @@
+---
+"@digdir/designsystemet": patch
+---
+
+tokens: fix wrong reference for focus color. Now uses neutral color
```

Comment:
Should this changeset be for css as well?

Reply:
its all built from tokens so unsure. I can try rephrasing it.

Reply:
If it's present/used in our css, I think it should be there as well

Reply:
Its part of the theme thats in the package or users generate themselves